src/core/qgis.h

```diff
@@ -5983,6 +5983,24 @@ class CORE_EXPORT Qgis
     };
     Q_ENUM( StacObjectType )
 
+    /**
+     * Capabilities of a raster layer processing parameter.
+     * \since QGIS 4.0
+     */
+    enum class RasterProcessingParameterCapability : int SIP_ENUM_BASETYPE( IntFlag )
+    {
+      WmsScale = 1 << 0, //!< The parameter supports a reference scale
+      WmsDpi = 1 << 1,   //!< The paramenter supports a server resolution (e.g., for a WMS server)
```

Comment:
```suggestion
      WmsDpi = 1 << 1,   //!< The parameter supports a server resolution for WMS source layers
```
(or rename the enum value itself to something non-wms specific)